website/transformDocs.js

```diff
@@ -79,7 +79,12 @@ const groupSort = (g1, g2) => {
 function getGroupName(object) {
     const groupPredicates = {
         'Errors': (x) => x.name.toLowerCase().includes('error'),
-        'Main Classes': (x) => ['Dataset', 'KeyValueStore', 'RequestQueue'].includes(x.name) || x.name.endsWith('Crawler'),
+        'Main Classes': (x) => [
```

Comment:
Wow this is crazy. Not relevant to the PR, I just needed to let my feelings out.